src/index.js

```diff
@@ -2,0 +1,43 @@
+// 'use strict';
+
+// import express from 'express';
+// import cors from 'cors';
+
+// const app = express();
+
+// app.use(cors({ origin: '*' }));
+// app.use(express.json());
+
+// const messages = [];
+
+// app.get('/', (req, res) => {
+//   res.send('Hello World');
+// });
+
+// app.post('/message', (req, res) => {
+//   const { author, text } = req.body;
+
+//   if (!author || !text) {
+//     return res.status(400).send({ error: 'Author and text are required' });
+//   }
+
+//   const message = {
+//     author,
+//     text,
+//     time: new Date(),
+//   };
+
+//   messages.push(message);
+
+//   res.status(201).send(message);
+// });
+
+// app.get('/messages', (req, res) => {
+//   res.status(200).send(messages);
+// });
+
+// const PORT = 3001;
```

Comment:
The definition of the server's port is commented out. Uncomment this line to set the port for the server.